crates/driver/src/tests/setup/blockchain/mod.rs

```diff
@@ -15,6 +19,17 @@ pub fn web3(url: &str) -> DynWeb3 {
     ))
 }
 
+/// Compute the effective gas price estimate from a node.
+pub async fn effective_gas_price(web3: &DynWeb3) -> eth::U256 {
+    let estimate = NativeGasEstimator::new(web3.transport().clone(), None)
+        .await
+        .unwrap()
+        .estimate()
+        .await
+        .unwrap();
+    eth::U256::from_f64_lossy(estimate.effective_gas_price())
```

Comment:
This could be done in a lossless manner, no?
I don't expect it to matter but it looks surprising.
Also why does this return `eth::U256` when the type `EffectiveGasPrive` already exists?

Reply:
This is in the testing code. I wanted to not use domain types here.